content/master/concepts/claim-server-side-apply.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Claim Server-Side Apply
+state: alpha
+alphaVersion: "1.15"
+weight: 61
+---
+
+When you create a claim, Crossplane creates a corresponding composite resource
+(XR). Crossplane keeps the claim in sync with the composite resource. When you
+change the claim, Crossplane reflects those changes on the composite resource.
+
+Read the [claims documentation]({{<ref "./claims">}}) to learn more about claims
+and how they relate to composite resources.
+
+Crossplane can use server-side apply to keep the claim in sync with the
+composite resource.
+
+{{<hint "tip">}}
+Use the `--enable-claim-ssa` feature flag to enable using server-side apply.
+
+Read the
+[Install Crossplane documentation]({{<ref "../software/install#feature-flags">}})
+to learn about feature flags.
+{{</hint>}}
+
+Server-side apply is a Kubernetes feature. When Crossplane uses server-side
+apply, the Kubernetes API server helps sync claims with composite resources.
+This fixes several bugs. Read more about server-side apply in the
+[Kubernetes documentation](https://kubernetes.io/docs/reference/using-api/server-side-apply/).
+
+
+If you see unexpected behavior when Crossplane syncs a claim with a composite
+resource, test whether enabling server-side apply fixes the problem. Server-side
+apply fixes the following problems:
+
+- Fields reappearing after you delete them from a claim's spec.
+- Crossplane not deleting labels and annotations from the composite resource
+  when you delete them from the claim.
+
+{{<hint "important">}}
+When you enable server-side apply, Crossplane is more strict about how it syncs
+claims with XRs:
+
+- The claim's `metadata` syncs to the composite resource's `metadata`.
+- The claim's `spec` syncs to the composite resource's `spec`.
+- The composite resource's `status` syncs to the claim's `status`.
+
+When you enable server-side apply Crossplane doesn't sync the composite resource's `metadata`
+and `spec` back to the claim's `metadata` and `spec`. It also doesn't sync the
+claim's `status` to the composite resource's `status`.
+{{</hint>}}
+
+## Use claim server-side apply with composition functions
+
+When you use composition functions, Crossplane also uses server side apply to
+sync composite resources with composed resources. Read more about this in the
+[composition functions documentation]({{<ref "./composition-functions#how-composition-functions-work">}}).
+
+```mermaid
+graph LR
+  A(Claim) -- claim server-side apply --> B(Composite Resource)
+  B -- function server-side apply --> C(Composed Resource)
+  B -- function server-side apply --> D(Composed Resource)
+  B -- function server-side apply --> E(Composed Resource)
+```
+
+{{<hint "tip">}}
+When Crossplane uses server-side apply to sync claims with composed resources,
```

Comment:
> When Crossplane uses server-side apply to sync claims with composed resources,

Do you mean here actually "to sync composite with composed resources"?

`--enable-claim-ssa` is not about merging complex fields - given that Crossplane owns both the claim and the composite, it ensures just the proper propagation of claim's changes down to the counterpart composite.

Merging complex fields are interesting when several different actors are owning a part of it (e.g. an item in the list or a field in an object). Such situations can appear for composite to composed syncs.

Reply:
> Do you mean here actually "to sync composite with composed resources"?

I mean to sync claims with composed resources. I'm intentionally omitting the fact that there's actually an XR in the middle, to avoid going into too much detail. My hope is that the diagram above, which shows the XR in the middle, would make this apparent.

I understand that claim level SSA does not in and of itself help with merging fields. The point I'm trying to make is this one: https://github.com/crossplane/docs/pull/711/files#r1491652688. I think my point is not coming across. Let me try to rewrite this section.